JobConfig/primary/prolog.fcl

```diff
@@ -153,16 +153,16 @@ Primary: {
     }
 
     GammaConversionFilter : {
-	module_type : GammaConversionPoints
-	g4ModuleLabel : "g4run"
-	doFilter : true
-	rMin : 0.   # Include target and IPA
-	rMax : 500.
-	zMin : 5000.
-	zMax : 8000.
-	eMin : 1.5
-	verbosity : 0
-	saveTree : false
+      module_type : GammaConversionPoints
+      g4ModuleLabel : "g4run"
+      doFilter : true
+      rMin : 0.   # Include target and IPA
+      rMax : 500.
+      zMin : 5000.
+      zMax : 8000.
+      eMin : 1.5
+      verbosity : 0
+      saveTree : true
```

Comment:
Is there a reason for turning back on this ROOT TTree output? My previous commit turned this off as it was only on for validation and I didn't intend to push it with it on.

Reply:
This may have gotten lost in the merge conflict resolutions. Let me look...

Reply:
This was flagged only as a change in indenting: the value of saveTree and all the related parameters are unchanged compared to your previous commit.

Reply:
Perhaps this is an issue of commit order? Here's the value at the HEAD: https://github.com/Mu2e/Production/blob/main/JobConfig/primary/prolog.fcl#L165
Could it be worth rebasing to check that the final value will be `false`?

Reply:
I see that you resolved this either way, thank you!